www/dashboard/nmasspays.spt

```diff
@@ -0,0 +1,16 @@
+[--------]
+nmasspays = website.db.one("""
+
+    WITH payday AS (SELECT ts_start FROM paydays ORDER BY ts_start DESC LIMIT 1)
+
+    SELECT count(*)
+      FROM exchanges e
+      JOIN exchange_routes er
+        ON e.route = er.id
+     WHERE er.network = 'paypal'
+       AND e.amount < 0
```

Comment:
The semantics of the `exchanges.amount` field are somewhat goofy (especially as it interacts with the `fee` field; see #2442), but fundamentally a negative amount indicates a payout, whereas a positive amount indicates a payin.